examples/cpp/Makefile

```
@@ -1,4 +1,4 @@
-MOLE = ../../src/cpp
+MOLE = ../../src/mole_C++
```

Hi @Amey-kulkarni19 , the line above should not have changed. The actual path on `master` is `src/cpp`, so you should have kept 

```suggestion
MOLE = ../../src/cpp
```

---

Oh!

I don't know how it changed. But I have corrected it.